modules/hostap/CMakeLists.txt

```diff
@@ -215,8 +215,8 @@ zephyr_library_compile_definitions_ifdef(CONFIG_WIFI_NM_WPA_SUPPLICANT_AP
 	NEED_AP_MLME
 	CONFIG_IEEE80211AC
 	CONFIG_IEEE80211AX
-	CONFIG_EAP_SERVER
-	CONFIG_EAP_SERVER_IDENTITY
+	EAP_SERVER
+	EAP_SERVER_IDENTITY
```

Comment:
better to use a separate commit to fix this.

Reply:
yes, will separate